idaes/surrogate/plotting/sm_plotter.py

```diff
@@ -30,22 +30,30 @@
 """
 
 
-def scatter2D(xdata, zdata, xtest, zfit, xlabels=None, zlabels=None, dz=None,
-              dzfit=None, clo=None, chi=None, clabel=None, show=True,
-              PDF=False, filename='results_scatter2D.pdf'):
+def surrogate_scatter2D(surrogate, dataframe, filename=None):
+    input_data = dataframe[surrogate.input_labels()]
+    output_data = dataframe[surrogate.output_labels()]
+    output_surrogate = surrogate.evaluate_surrogate(input_data)
+    _scatter2D(xdata=input_data.values,
+               zdata=output_data.values,
+               zfit=output_surrogate.values,
+               xlabels=surrogate.input_labels(),
+               zlabels=surrogate.output_labels(),
+               filename=filename)
+
+
+def _scatter2D(xdata, zdata, zfit, xlabels=None, zlabels=None,
+               show=True, filename=None):
```

Comment:
Comment for discussion - the private method has a `show` argument, but the public method does not. Should we include the `show` argument in the public method as well to allow for cases where the user would prefer to just save the plots to file before looking at them?

Reply:
We could....I'll leave the default to 'True' since most users would want the plots to print to their IDE viewer.